### PR TITLE
MonadReader instace for the SqlQueryT

### PR DESCRIPTION
**Pull request by ikv**

### CODE REVIEW

File: src/Database/Persist/Monad.hs [hunk 1]
```diff
@@ -193,6 +197,13 @@ newtype SqlQueryT m a = SqlQueryT
     , MonadMask
     )
 
+mapSqlQueryT :: (m a -> n b) -> SqlQueryT m a -> SqlQueryT n b
+mapSqlQueryT f m = SqlQueryT . ReaderT $ f . runReaderT (unSqlQueryT m)
```
Comment (brandonchinn178) [inline src/Database/Persist/Monad.hs]: can you use `mapReaderT` here instead?

File: src/Database/Persist/Monad.hs [hunk 2]
```diff
@@ -193,6 +197,13 @@ newtype SqlQueryT m a = SqlQueryT
...
+
+instance MonadReader r m => MonadReader r (SqlQueryT m) where
```
Comment (brandonchinn178) [inline src/Database/Persist/Monad.hs]: can you move this below the `MonadUnliftIO` instance?

**Review by brandonchinn178 — COMMENTED** [review]
Thanks! Just some small comments

**Comment by ikv** [comment]
Okay, how do i make codecov happy?

**Review by brandonchinn178 — APPROVED** [review]

**Comment by brandonchinn178** [comment]
Don't worry about codecov. It's just complaining because you added more lines of code, meaning the denominator is bigger, making the ratio smaller. It's just there to help eyeball coverage changes.

**Pull request merged**